holoviews/core/operation.py

```diff
@@ -197,16 +197,18 @@ def __call__(self, element, **params):
                     isinstance(element, DynamicMap))
                    or self.p.dynamic is True)
 
-        if isinstance(element, ViewableElement):
-            processed = self._process(element)
-        elif isinstance(element, GridSpace):
+        if isinstance(element, GridSpace):
             # Initialize an empty axis layout
             grid_data = ((pos, self(cell, **params))
                          for pos, cell in element.items())
             processed = GridSpace(grid_data, label=element.label,
                                   kdims=element.kdims)
         elif dynamic:
-            processed = DynamicFunction(element, function=self, kwargs=params)
+            streams = getattr(self, 'streams', [])
```

Comment:
This feature probably needs mentioning in the docstring wherever ``dynamic`` is mentioned.